social_django/strategy.py

```diff
@@ -114,13 +114,6 @@ def authenticate(self, backend, *args, **kwargs):
         kwargs['backend'] = backend
         return authenticate(*args, **kwargs)
 
-    def clean_authenticate_args(self, request=None, *args, **kwargs):
```

Comment:
Removing is not an option since starting with Django 2.0 the positional argument is a requirement, a better cleanup is needed instead to avoid the double arguments passing.